web/Views/Property/_AddFormStep5.cshtml

```diff
@@ -25,11 +25,11 @@
     <label class="form-label">Do you accept pets?</label>
 
     <div class="form-check">
-        <input type="radio" asp-for="AcceptsPets" value="true" id="PetsTrue" class="form-check-input">
+        <input type="radio" asp-for="AcceptsPets" value="true" id="PetsTrue" class="form-check-input" required>
         <label for="PetsTrue" class="form-check-label">Yes</label>
     </div>
     <div class="form-check">
-        <input type="radio" asp-for="AcceptsPets" value="false" id="PetsFalse" class="form-check-input">
+        <input type="radio" asp-for="AcceptsPets" value="false" id="PetsFalse" class="form-check-input" required>
         <label for="PetsFalse" class="form-check-label">No</label>
     </div>
 </div>

```

Comment:
Maybe a similar check should be applied to the fields 'AcceptsSingleTennant', 'AcceptsCouple' and 'AcceptsFamily' - something which requires at least one option (although possibly more than one option, unlike here) to be selected? It's currently possible to leave all of these unselected, which could lead to landlords accidentally providing information (i.e. by misreading the form as 'Not accept' or something) which says they won't accept any tenants.

Reply:
Good idea, I'm on it

Reply:
That's a good point, we should definitely implement a check.

Might be easier to do after splitting out the pages to have separate view models though if it's difficult to do now..

Reply:
All done :)